class-wc-calypso-bridge-dotcom-features.php

```diff
@@ -103,18 +169,120 @@ public static function has_ecommerce_features() {
 	}
 
 	/**
-	 * Determine if site is Ecommerce and cache it.
+	 * Determine if site is a paid Ecommerce plan and cache it (refers all ecommerce plans from dotCom or WCCOM).
 	 *
 	 * @var bool
 	 */
 	public static function is_ecommerce_plan() {
 		if ( is_null( self::$is_ecommerce_plan ) ) {
-			self::$is_ecommerce_plan = self::has_ecommerce_features() && wpcom_site_has_feature( \WPCOM_Features::INSTALL_PLUGINS );
+			self::$is_ecommerce_plan = self::has_ecommerce_features() && (
+				wpcom_site_has_feature( \WPCOM_Features::ECOMMERCE_MANAGED_PLUGINS_SMALL ) || wpcom_site_has_feature( \WPCOM_Features::ECOMMERCE_MANAGED_PLUGINS_MEDIUM )
+			);
 		}
 
 		return self::$is_ecommerce_plan;
 	}
 
+	/**
+	 * Determine if site is Ecommerce from dotCom and cache it.
```

Comment:
Another `dotCom` => `WordPress.com`
```suggestion
	 * Determine if site has a WordPress.com eCommerce plan and cache the result.
```